docs/internal-developers/block-client-apis/checkout/checkout-api.md

```diff
@@ -6,7 +6,7 @@
 
 -   [Data Stores](#data-stores)
     -   [Checkout Data Store](#checkout-data-store)
-        -   [Selectors](#selectors)
+        -   [Selectors](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/data-store/checkout.md#selectors)
```

Comment:
What are your thoughts on using relative links instead of absolute links? In the other docs, we're also using relative links. When merging WooCommerce Blocks into core, we won't have to adjust the links then.

Reply:
Good point, this will break like next week.